build/scripts/build-inspector.sh

```diff
@@ -0,0 +1,56 @@
+#!/usr/bin/env bash
+
+set -e
+
+# TODO: Add as CMake Target
+
+WORKSPACE=`pwd`
+CONFIGURATION="Release"
+
+mkdir -p "$WORKSPACE/cmake-build"
+
+WEBKIT_SOURCE_PATH="$WORKSPACE/src/webkit"
+WEBKIT_BUILD_OUTPUT_PATH="$WORKSPACE/cmake-build/WebKit-Xcode"
+
+INSPECTOR_SOURCE_PATH="$WORKSPACE/src/debugging/Inspector/Inspector"
+INSPECTOR_BUILD_OUTPUT_PATH="$WORKSPACE/cmake-build/Inspector"
+
+echo "Building WebKit..."
+xcodebuild \
+    -workspace "$WEBKIT_SOURCE_PATH/WebKit.xcworkspace" \
+    -configuration "$CONFIGURATION" \
+    -scheme "All Source" \
+    SYMROOT="$WEBKIT_BUILD_OUTPUT_PATH" \
+    build \
+    >> "$WORKSPACE/build.log" 2>&1
+
+echo "Copying frameworks..."
+rm -rf "$INSPECTOR_SOURCE_PATH/Frameworks"
+find "$WEBKIT_BUILD_OUTPUT_PATH/$CONFIGURATION" -name "*.framework" -type d -maxdepth 1 -print \
+    -exec rsync -a {} "$INSPECTOR_SOURCE_PATH/Frameworks/" \; \
+    >> "$WORKSPACE/build.log" 2>&1
+
+echo "Building Inspector app..."
+# TODO: Code signing
+xcodebuild \
+    -project "$INSPECTOR_SOURCE_PATH/Inspector.xcodeproj" \
+    -configuration "$CONFIGURATION" \
+    -scheme "Inspector" \
+    SYMROOT="$INSPECTOR_BUILD_OUTPUT_PATH" \
+    build \
+    >> "$WORKSPACE/build.log" 2>&1
+
+echo "Packaging Inspector app..."
+pushd $INSPECTOR_BUILD_OUTPUT_PATH/$CONFIGURATION/ \
+    >> "$WORKSPACE/build.log" 2>&1
+rm -f "NativeScript Inspector.zip"
+rm -rf "NativeScript Inspector.app" "NativeScript Inspector.app.dSYM"
+mv "Inspector.app" "NativeScript Inspector.app"
```

Comment:
Can someone verify these lines here. The new Inspector archive seems to have the frameworks duplicated.